react/ssr-app/vite.config.ts

```diff
@@ -8,4 +8,8 @@ export default defineConfig((config) => ({
 	resolve: {
 		alias: config.mode === 'production' ? {} : alias,
 	},
+	define: {
+		// make sure we always have react warnings even on the CI with "npm run preview"
+		'process.env.NODE_ENV': '"development"',
```

Comment:
I would rather remove this line.
In our CI, we want the e2e test of the react ssr app to run targetting a normal ssr react app...
For example, do you know if this also affects React development double-rendering ?

Reply:
yes, I think this also affects React development double-rendering
I think it is important not to neglect the development mode, there are more checks than in production mode and it does not look good if using AgnosUI in development brings a lot of errors that we do not detect in our test because we test in production mode.

Reply:
This is an interesting topic for sure, which could also be applied to the normal e2e tests, not just the ssr !
I opened the issue #315 some time ago so that we could one day properly avoid any error / warning, but it's outside the scope of this ssr react PR.

For this PR, I would propose either only production or both modes (as most dev-mode warnings would not be discovered by us anyway).